### ISSUE TITLE
Revery roadmap in face of Reason->Rescript rebrand

### ISSUE THREAD

**Issue by marcinkoziej** [revery-ui/revery#1028]
Hi!
Amazing Work!!!
What are you planning to do about ReasonML depreciation in favor of Rescript?

**Comment by lessp** [comment]
It seems as there's still quite a lot of confusion regarding Reason/ReScript!

ReScript branched out from Reason, so instead of simply being an alternative syntax and toolchain to OCaml, ReScript decided to put all their focus on JS and to own the toolchain. There was already some fraction happening since BuckleScript (now ReScript) is on an old version of OCaml, so while people in the native world were frustrated with BuckleScript lagging behind, people behind BuckleScript had their own concerns. 

Reason on the other hand hasn't changed, apart from the fact that Reason is also getting an updated syntax (which closely resembles the changes to the ReScript-syntax), it's still an alternative syntax and toolchain to OCaml.

Revery, being native is not really affected by these changes at all apart from the added confusion!

tl;dr

**ReScript:** Only focuses on compiling to JS
**Reason:** Still an alternative syntax/toolchain to OCaml, uses the OCaml-platform and can compile to JS (via BuckleScript or js_of_ocaml), Native and Bytecode.

-----------

![image](https://user-images.githubusercontent.com/17602389/102764418-c59b5400-4383-11eb-9ad9-0cfb5cd8ec8b.png)

**Comment by marcinkoziej** [comment]
Thank You for this explanation!
